src/tangojs-connector-mtango/MTangoConnector.js

```diff
@@ -247,13 +251,20 @@ export class MTangoConnector extends tangojs.Connector {
    */
   write_device_attribute (devname, attrs) {
 
-    // PUT http://localhost:8080/mtango/rest/rc2/devices/sys/tg_test/1/attributes/long_scalar?value=xx
+    // PUT http://localhost:8080/mtango/rest/rc3/hosts/localhost/10000/devices/sys/tg_test/1/attributes/values?long_scalar=xx
 
-    const nvList = attrs.map(a => [a.name, a.value])
-
-    return Promise.all(nvList.map(([name, value]) => {
-      this._fetch('put', `devices/${devname}/attributes/${name}?value=${value}`)
-    }))
+    const nvList = attrs.map(a => `${a.name}=${a.value}`).join('&')
+  return this._fetch('put', `devices/${devname}/attributes/value?${nvList}`)
+      .then(values => values.map( value => {
+          return new tangojs.api.DeviceAttribute(Object.assign(value, {
+            quality: value.quality,
```

Comment:
You should use `normalizeAttrQuality` here to get `AttrQuality` object instead of a string